gradle.properties

```diff
@@ -15,4 +15,8 @@
 # When configured, Gradle will run in incubating parallel mode.
 # This option should only be used with decoupled projects. More details, visit
 # http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
-# org.gradle.parallel=true
\ No newline at end of file
+# org.gradle.parallel=true
+
+# When set to true the Gradle daemon is used to run the build. For local developer builds this is our favorite property.
+# The developer environment is optimized for speed and feedback so we nearly always run Gradle jobs with the daemon.
+org.gradle.daemon=true
```

Comment:
We don't have an automated build for this, but according to the [Gradle docs](https://docs.gradle.org/current/userguide/gradle_daemon.html), we should disable the Gradle daemon for CI builds. There's probably a way for developers to use the daemon on local builds, but disable the daemon on CI builds. Not sure how that's done.